### PR TITLE
Revised UI tweaks

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by courville** [comment]
Thanks for the PR and the revised commits with more detailed on what it is intended to.
I think these are great additions on the leanback (Android TV) UI.
The only issue I have for now, is that I would like to preserve a parity in the UI look at feel between the phone/tablet and TV.
Would you feel ok to propagate the changes also in the non-leanback part of the code?

**Comment by ghost** [comment]
I do not really use the non-leanback UI. I looked a bit but it is very different.
Here are some new changes.

File: src/main/java/com/archos/mediacenter/video/player/PlayerController.java
Comment (courville) [inline src/main/java/com/archos/mediacenter/video/player/PlayerController.java:1269]: I prefer preserving the current view since it is explicit enough to have the remaining time on the right and there is no such thing that negative time...

**Review by courville — APPROVED** [review]

File: src/leanback/java/com/archos/mediacenter/video/leanback/presenter/ShortcutDetailsPresenter.java
Comment (courville) [inline src/leanback/java/com/archos/mediacenter/video/leanback/presenter/ShortcutDetailsPresenter.java]: perhaps better to remove specifically "/" instead of the systematically the last char

Comment (courville) [reply src/leanback/java/com/archos/mediacenter/video/leanback/presenter/ShortcutDetailsPresenter.java]: Need to check if all the shortcuts smb/upnp/sftp etc.. has an ending /

File: res/values/strings.xml
Comment (courville) [inline res/values/strings.xml]: we prefer info

File: res/xml/preferences_video.xml
Comment (courville) [inline res/xml/preferences_video.xml:223]: I prefer to keep is true by default

**Comment by courville** [comment]
Just saw your discussions on irc channel, please do not force push any change on the current PR.
I am git cherry-picking into nova all the changes that we find ok (we are doing code review with Phie).
Once merged let's treat the other as new individual PR.
Drop in on irc fore more if need be.

File: res/layout/card_layout.xml
Comment (courville) [inline res/layout/card_layout.xml:55]: I think this is a nice idea to populate the info card with more valuable information but I would add more info on codecs first then hwdec or swdec without details.

File: res/xml/preferences_video.xml
Comment (courville) [inline res/xml/preferences_video.xml:96]: I will not merge this change since the setting applies both to AndroidTV and phones/tablets. This is thus not intended to TV only.

**Comment by courville** [comment]
I found a way to bundle your commits using git rebase -i and the squash functionality.
It will look better on git history.
However it requires me to push and rewrite github git history.
This means that you will have to do a full repo init afterwards to get in sync otherwise there will be issues.
If this is ok, I will do it tonight.

**Comment by courville** [comment]
@persi-persu @PhieF @phhusson @av500 @nnoury I forced pushed my current Video git repo to github where I squashed may commits to look good. This will interfere with your current already pulled versions.
Be warned ;-)

**Pull request closed without merging**
